cypress/integration/calculation-triggered-when-unhiding-field.test.js

```diff
@@ -0,0 +1,30 @@
+import {
+    visitPage,
+    cfFieldIsNotVisible,
+	cfFieldIsVisible,
+	cfFieldSetValue,
+	cfFieldCheckValue,
+    cfFieldCheckAllValues
+} from '../support/util';
+
+describe('Calculation value is showed when unhiding the calculation field', () => {
```

Comment:
@New0 These tests show that the bug is fixed. The comment I left on the change is a concern that is neither proven or unproven. Can you please add a second test case here with a form whose calculation field starts NOT hidden, then is hidden and then unhidden? That's good coverage to have by itself. If those tests pass then ignore my other comment.